ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -1,5 +1,40 @@
 <template>
-  <v-container fluid no-gutters class="white pa-6">
+  <v-container fluid no-gutters class="white pa-0"  v-if="summaryView">
+    <v-card flat id="length-trust-summary">
+      <v-row no-gutters :class="[$style['length-trust-summary-header'], pa-2]">
+        <v-col cols="auto">
+          <v-icon color="#38598A">mdi-calendar-clock</v-icon>
+          <label class="length-trust-title pl-3"><strong>Registration Length and Trust Indenture</strong></label>
```

Comment:
is the `length-trust-title` css actually getting applied here? If so I guess it's because its nested under `length-trust-summary-header` or did you find another way to avoid the `$style` notation?

Reply:
Updated, removed length-trust-title as it's only padding left. I started with a create confirm template and this came with it.